src/fusion.cpp

```diff
@@ -205,11 +205,13 @@ miopenStatus_t FusionPlanDescriptor::GetWorkspaceSizeImmed(Handle& handle,
     {
         if(op->kind() == miopenFusionOpConvForward)
         {
-            auto ptr = std::dynamic_pointer_cast<ConvForwardOpDescriptor>(op);
+            auto& ptr = dynamic_cast<ConvForwardOpDescriptor&>(*op);
```

Comment:
@DrizztDoUrden It is not a pointer anymore, right?

Reply:
Will rename